libcontainer/apparmor/apparmor_unsupported.go

```diff
@@ -6,13 +6,14 @@ import (
 	"errors"
 )
 
+// ErrApparmorNotEnabled indicates that AppArmor is not enabled or not supported.
 var ErrApparmorNotEnabled = errors.New("apparmor: config provided but apparmor not supported")
```

Comment:
This looks defined twice (but passing CI?)

Reply:
Good catch; looks like I didn't remove it here.

I guess CI doesn't catch it as it only runs on Linux?

Reply:
Fixed (removed this one)